core/domain/stats_services.py

```diff
@@ -18,23 +18,166 @@
 
 import itertools
 
-from core.domain import exp_services
+from core.domain import exp_domain
 from core.domain import interaction_registry
 from core.domain import stats_domain
-from core.domain import stats_jobs_continuous
 from core.platform import models
 
 (stats_models,) = models.Registry.import_models([models.NAMES.statistics])
 
+# Counts contributions from all versions.
+VERSION_ALL = 'all'
+
+def handle_stats_creation_for_new_exploration(exp_id, exp_version, state_names):
+    """Creates ExplorationStatsModel for the freshly created exploration and
+    sets all initial values to zero.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version. int. Version of the exploration.
+        state_names: list(str). State names of the exploration.
+    """
+    state_stats_mapping = {
+        state_name: stats_domain.StateStats.create_default()
+        for state_name in state_names
+    }
+
+    exploration_stats = stats_domain.ExplorationStats(
+        exp_id, exp_version, 0, 0, state_stats_mapping)
+    create_stats_model(exploration_stats)
+
+
+def handle_stats_creation_for_new_exp_version(exploration, change_list):
```

Comment:
@seanlip Let's change this parameter to exp_id, exp_version, state_names as well?

Reply:
Yep, sg

Reply:
Yep, sounds good -- let's do that.

Reply:
Done.